scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -887,6 +896,33 @@ class MicrobeEditor{
         return (playerSpecies.generation+1);
     }
 
+    void getNucleus()
```

Comment:
This should have a cleaner name. Also the nucleusIsPresent property should *really* be a boolean.